tests/unit_tests/test_wave_06.py

```diff
@@ -109,16 +108,15 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_a and item_b and item_f in tai.inventory
+    assert item_c in jesse.inventory 
+    assert item_d and item_e in jesse.inventory
```

Comment:
You can combine lines 112 and 113 if you want:
assert len(tai.inventory) == 3 and len(jesse.inventory) == 3

You can use list destructing like this too - a pythonic way of doing it:

assert [item_a, item_b, item_f] == tai.inventory
assert [item_c, item_d, item_e] == jesse.inventory

https://medium.com/@umaramanat66/destructuring-list-in-python-like-javascript-f7d4c0968538